tasks_week03_Ruben_Chiche/include/Animal.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+#include <string>
+#include <vector>
+#include <iostream>
+#include "Location.h"
+
+using std::string;
+
+constexpr auto ROWS = 20;
+constexpr auto COLUMNS = 40;
+
+
+class Animal
+{
+public:
+	// constructor
+	Animal(const string&, const Location&);
+
```

Comment:
no use of virtual destructor 
read this
https://stackoverflow.com/questions/461203/when-to-use-virtual-destructors

Reply:
fixed